playground/mocks/data/idp/idx/error-device-code-activate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "stateHandle":"02itnqG312DoS3cU0z0LWs11l76yQ8ll4d95Oye61u",
+  "version":"1.0.0",
+  "expiresAt":"2020-04-13T20:30:53.000Z",
+  "step":"IDENTIFY",
+  "intent":"LOGIN",
+  "messages": {
+    "type": "array",
+    "value": [
+      {
+        "message": "Invalid code. Try again.",
+        "i18n": {
+          "key": "api.authn.error.PASSCODE_INVALID",
```

Comment:
there is no need to keep `PASSCODE_INVALID` as capitalized. possible to make that consistent with other error keys..?

Reply:
this key was already existing in v1 pipeline so I am not sure what other use cases are there. I can take a look in next pr.

Reply:
is this valid error key on backend `idx.invalid.device.code` .. i see this exist in widget.

Reply:
`idx.invalid.device.code` is not defined on backend. I think we used `api.authn.error.PASSCODE_INVALID` in both legacy and idx to keep it consistent. but happy to change it to `idx.invalid.device.code` for idx flow.

Reply:
It's okay to reuse existing. i just wanted to see if its on the backend.